docker-compose.yml

```diff
@@ -65,12 +65,12 @@ services:
       - "6379:6379"
 
   kafka:
-    image: wurstmeister/kafka:0.9.0.1-1
+    image: wurstmeister/kafka
     ports:
       - "9092:9092"
     environment:
-      #KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://localhost:9092    # If trying to connect from outside docker, enable this
-      #KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://kafka:9092    # If trying to connect from inside docker, enable this
+      KAFKA_LISTENERS: PLAINTEXT://:9092
+      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://192.168.1.198:9092
```

Comment:
Should we revert lines 72 and 73 back to the master config?

Reply:
Yes!